api/api_router.js

```diff
@@ -1,8 +1,10 @@
 const router = require('express').Router();
 
 const authRouter = require('../routes/auth_router');
+const userRouter = requre('../routes/user_router.js');
 
 router.use('/auth', authRouter);
+router.use('/users', userRouter);
```

Comment:
We can probably add the restricted middleware here instead of on the specific user routes